rds.py

```diff
@@ -37,6 +40,8 @@ def add_pretty_names(instances):
             extra = 'Double'
         elif short.startswith('10x'):
             extra = 'Deca'
+        elif short.startswith('16x'):
+            extra = 'Hexa'
```

Comment:
Does AWS use this terminology? Hexa should mean 6 like in hexagon: http://phrontistery.info/numbers.html

Reply:
I used it as a shortcut for Hexadecimal. How about "Hexadecimal" or "16"?
BTW - can we drop the "Warning: RDS pricing information is often inaccurate. See issue #314." notification?

Reply:
I think "16" would be better. Looking at the site makes me think it might be nice to switch some of the others to simple numbers as well so they're easier to read and alphabetically sort well. Another option is just to remove this change from this PR since it's not related to the pricing fix. If you do leave it in, can you mirror it to `scrape.py` as well?

And yeah - removing that notice would be great! Your fix here is greatly appreciated.